.github/workflows/cli.yml

```diff
@@ -0,0 +1,132 @@
+name: CLI Tests
+
+on:
+  push:
+    branches: [master, main, force_test, release-*]
+    tags: ['*']
+  pull_request:
+    branches: ['**']
+
+defaults:
+  run:
+    working-directory: ./cli
+
+jobs:
+  build:
+    name: Build and Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Import environment variables
+        run: cat ".github/env" >> $GITHUB_ENV
+
+      - name: Set up Go ${{ env.golang-version }}
+        uses: actions/setup-go@v2.1.5
+        with:
+          go-version: ${{ env.golang-version }}
+        id: go
+
+      - name: Format
+        run: |
+          make go-fmt
+
+      - name: Vet
+        run: |
+          make go-vet
+
+      - name: Lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: ${{ env.golangci-lint-version }}
+
+      - name: Build
+        run: |
+          make build
+
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Import environment variables
+        run: cat ".github/env" >> $GITHUB_ENV
+
+      - name: Set up Go ${{ env.golang-version }}
+        uses: actions/setup-go@v2.1.5
+        with:
+          go-version: ${{ env.golang-version }}
+
+      - name: Test static generation
+        run: |
+          ./generate-deploy-script.sh
+          # git diff --exit-code
```

Comment:
why is this commented?

Reply:
Good question :thinking: This part is unchanged from current workflow and can be seen in https://github.com/timescale/tobs/blob/master/.github/workflows/generate.yml#L27. I'll check if this needs to be reenabled or removed.

Reply:
Turns out this should've been enabled and we had incorrect `deploy/static/deploy.yaml` file because of it. I reenabled this part and regenerated the file.